### PR TITLE
Create june-july-invite.md

### PR DESCRIPTION
**Pull request by raynamharris**
sending this as a test invite to team copper first

### CODE REVIEW

File: draft-emails/june-july-invite.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+We are excited to invite you to to the next two monthly workshops! 
+
+Here are the links to the workshop websites with (hopefully) all the details you need and want!
```
Comment (charlesreid1) [inline draft-emails/june-july-invite.md]: no (hopefully)

File: draft-emails/june-july-invite.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+-  June 29-30 in Portland, OR: http://nih-data-commons.us/2018-june-workshop/
+-  July 25-26 in Chapel Hill, NC: http://nih-data-commons.us/2018-july-workshop/ 
+
+_Wondering if you should attend these monthly meetings?_ 
+Feedback from the May post-workshop survey confirmed that these meetings
+extremely are useful for discussing details, obstacles, and goals in a smaller setting.
```
Comment (charlesreid1) [inline draft-emails/june-july-invite.md]: "extremely are" -> "are extremely"

File: draft-emails/june-july-invite.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Feedback from the May post-workshop survey confirmed that these meetings
+extremely are useful for discussing details, obstacles, and goals in a smaller setting. 
+Your fellow DCPPC colleagues recommend that 
+_"each team to bring their developers"_ and that 
+_"everyone from each team should attend at least once"_. 
+
+Also, Team Copper is covering all your travel expenses and 
+is incorporating your excellent suggestions to improve the quality and utility of these workshops. 
+
+We would love to see 3-6 members from each Team and Data Steward at both meetings.
```
Comment (charlesreid1) [inline draft-emails/june-july-invite.md]: "from each" -> "representing each"

File: draft-emails/june-july-invite.md (hunk 1 above)
Comment (raynamharris) [reply draft-emails/june-july-invite.md]: incorporated

File: draft-emails/june-july-invite.md (hunk 2 above)
Comment (raynamharris) [reply draft-emails/june-july-invite.md]: changed

File: draft-emails/june-july-invite.md (hunk 3 above)
Comment (raynamharris) [reply draft-emails/june-july-invite.md]: yep. great suggestion

**Comment by charlesreid1** [comment]
Do you want to wait for more feedback or is this ready to merge?

**Pull request merged**